android/app/src/androidTest/java/host/exp/exponent/TestSuiteTests.kt

```diff
@@ -0,0 +1,151 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+package host.exp.exponent
+
+import android.Manifest
+import android.content.Intent
+import android.net.Uri
+import androidx.test.InstrumentationRegistry
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.IdlingResource
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.rule.GrantPermissionRule
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.Until
+import host.exp.exponent.annotations.ExpoAlwaysPassThroughFilter
+import host.exp.exponent.annotations.ExpoSdkVersionTest
+import host.exp.exponent.annotations.ExpoTestSuiteTest
+import host.exp.exponent.generated.ExponentBuildConstants
+import host.exp.exponent.kernel.KernelConfig
+import host.exp.exponent.utils.*
+import org.json.JSONException
+import org.json.JSONObject
+import org.junit.*
+import org.junit.rules.RuleChain
+import org.junit.runner.RunWith
+import org.junit.runners.MethodSorters
+
+@RunWith(ExpoTestRunner::class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+class TestSuiteTests : BaseTestClass() {
+  private lateinit var loadingScreenIdlingResource: IdlingResource
+  private lateinit var elapsedTimeIdlingResource: ElapsedTimeIdlingResource
+  private lateinit var jsTestRunnerIdlingResource: JSTestRunnerIdlingResource
+
+  @Before
+  fun before() {
+    loadingScreenIdlingResource = LoadingScreenIdlingResource()
+    elapsedTimeIdlingResource = ElapsedTimeIdlingResource()
+    jsTestRunnerIdlingResource = JSTestRunnerIdlingResource()
+    Espresso.registerIdlingResources(
+      loadingScreenIdlingResource,
+      elapsedTimeIdlingResource,
+      jsTestRunnerIdlingResource
+    )
+    KernelConfig.FORCE_UNVERSIONED_PUBLISHED_EXPERIENCES = false
```

Comment:
I wonder if this was a bug in the old code and should have been `true` given that it is an override